internal/provider/models/shared_source.go

```diff
@@ -88,6 +93,7 @@ func SharedSourceFromModel(plan *SharedSourceResourceModel) *SharedSource {
 // From an API response to a terraform model
 func SharedSourceToModel(plan *SharedSourceResourceModel, source *SharedSource) {
 	plan.Id = NewStringValue(source.Id)
+	plan.ConsumerId = NewStringValue(source.ConsumerId)
```

Comment:
Looks fine, but there should be at least 1 test updated to check this property addition

Reply:
My bad, tests cases updated.